### PR TITLE
feat: add QRCode feature

### PR DESCRIPTION
**Pull request by xfqz86**

### CODE REVIEW

File: niimblue/src/lib/QRCodeParamsControls.svelte [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      ><FaIcon icon="pen-to-square" /></span
+    >
+    <select class="from-select" value={ecl} on:change={eclChange}>
+      <option value="L">Level L</option>
+      <option value="M">Level M</option>
+      <option value="Q">Level Q</option>
+      <option value="H">Level H</option>
+    </select>
+  </div>
+  <textarea class="qrcode-content" value={text} on:change={textChange} />
```
Comment (MultiMote) [inline niimblue/src/lib/QRCodeParamsControls.svelte:49]: Add form-control class.

File: niimblue/src/lib/QRCodeParamsControls.svelte [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+{/if}
+
+<style>
+  .input-group {
+    width: fit-content;
+  }
+
+  .qrcode-content {
+    width: 100%;
```
Comment (MultiMote) [inline niimblue/src/lib/QRCodeParamsControls.svelte:58]: Remove (form-control is 100% width).

File: niimblue/src/fabric-object/qrcode.class.ts
Comment (MultiMote) [inline niimblue/src/fabric-object/qrcode.class.ts:1]: Is this intended to be so many typing check suppressions?

**Review by MultiMote — CHANGES_REQUESTED** [review]
Hi! Thank you so much for your contribution!
Please make a few changes so I can accept your pull request.

**Comment by MultiMote** [comment]
Also, saving then loading labels not working (`Cannot read properties of undefined (reading 'fromObject')`).

**Comment by xfqz86** [comment]
> Also, saving then loading labels not working (`Cannot read properties of undefined (reading 'fromObject')`).

[607c3d3](https://github.com/MultiMote/niimblue/pull/9/commits/607c3d3924d560715bc4ca5f573514a59f92d8df) fixed

**Review by MultiMote — APPROVED** [review]

**Pull request merged**